duplicacy/duplicacy_main.go

```diff
@@ -38,10 +38,14 @@ func getRepositoryPreference(context *cli.Context, storageName string) (reposito
 
     var pref_dir string = context.GlobalString("pref-dir")
     if pref_dir == "" {
-        repository, err := os.Getwd()
-        if err != nil {
-            duplicacy.LOG_ERROR("REPOSITORY_PATH", "Failed to retrieve the current working directory: %v", err)
-            return "", nil
+        repository = context.String("repository-dir")
+	    if repository == "" {
+	        var err error = nil // golang complains err is undefined if it isn't defined here
+		    repository, err = os.Getwd()
```

Comment:
Golang complains because you should've written `    repository, err = os.Getwd()`. I think you missed the `:` in `:=`

Reply:
Gah, yes. Thank you for noticing this. I must have dropped that during a copy/paste.